data/json/items/comestibles/raw_fruit.json

```diff
@@ -472,279 +472,352 @@
     "quench": 4,
     "calories": 30,
     "description": "A large purple plum.  Tasty and good for your digestion.",
-    "price": 200,
+    "price": 19,
     "price_postapoc": 100,
     "material": [ "fruit" ],
-    "volume": "66 ml",
+    "volume": "82 ml",
     "fun": 3,
     "flags": [ "SMOKABLE" ],
     "smoking_result": "dry_fruit",
-    "vitamins": [ [ "vitC", 8 ] ]
+    "vitamins": [ [ "vitC", 7 ] ]
   },
   {
     "type": "COMESTIBLE",
     "id": "grapes",
-    "name": { "str_sp": "grapes" },
-    "weight": "97 g",
+    "name": { "str": "handful of grapes", "str_pl": "handfuls of grapes" },
+    "weight": "92 g",
     "color": "light_green",
     "spoils_in": "5 days",
     "comestible_type": "FOOD",
     "symbol": "%",
     "quench": 5,
-    "calories": 165,
-    "description": "A cluster of juicy grapes.  Quite calorie-dense compared to most fruit, and you might be able to make them into wine if you're ambitious enough.",
-    "price": 75,
+    "calories": 61,
+    "description": "A handful of juicy grapes.  Quite calorie-dense compared to most fruit, and you might be able to make them into wine if you're ambitious enough.",
+    "price": 29,
     "price_postapoc": 100,
     "material": [ "fruit" ],
     "volume": "250 ml",
     "fun": 4,
     "flags": [ "FREEZERBURN", "SMOKABLE" ],
     "smoking_result": "dry_fruit",
-    "vitamins": [ [ "vitC", 4 ], [ "calcium", 1 ], [ "iron", 2 ] ]
+    "vitamins": [ [ "vitC", 4 ], [ "calcium", 1 ], [ "iron", 1 ] ]
   },
   {
     "type": "COMESTIBLE",
     "id": "pineapple",
     "name": { "str": "pineapple" },
-    "weight": "174 g",
+    "weight": "905 g",
     "color": "brown",
     "spoils_in": "3 days",
     "comestible_type": "FOOD",
     "symbol": "%",
     "quench": 9,
-    "calories": 87,
-    "description": "A large, spiky citrus fruit.  Delicious and a great source of Vitamin C, though a bit sour.",
-    "price": 450,
+    "calories": 452,
+    "description": "A large, spiky citrus fruit.  You reckon you should not try to eat it without further preparations.",
```

Comment:
A large, spiky citrus fruit.  A tough rind protects the tasty flesh inside, and digging into it as-is would be pretty inefficient and messy.  You should find a knife or something vaguely knife-like to cut it up with.